db/migrate/20220716113228_create_likes.rb

```diff
@@ -0,0 +1,5 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes, &:timestamps
+  end
+end
```

Comment:
- [ ] Nice job creating tables however this table should be associated with the user and post tables. Kindly use a migration like this :point_down

```.rbclass CreateLikes < ActiveRecord::Migration[7.0]
  def change
    create_table :likes do |t|
      t.references :author, foreign_key: { to_table: 'users' }
      t.references :post, foreign_key: true
      t.timestamps

    end
  end
end
```

Reply:
This issue is well resolved! ✔️